Scripts/Modules/TooltipModule/TooltipUI.cs

```diff
@@ -1,4 +1,5 @@
 ﻿#if UNITY_EDITOR
+using TMPro;
```

Comment:
Add CCU defines ad attribute, (see 6c7c8953)

Also, out of curiosity, why TMP_Text and not TextMeshProUGUI? Are we still using a canvas for tooltips?

Reply:
If canvas is not needed, better to just go with TMP_Text

Reply:
The canvas is needed for the icon.